spec/position_initializer_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+connect_to_sqlite
+
+describe RailsCoreExtensions::PositionInitializer, 'When repositioning' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.